README.md

```diff
@@ -41,6 +40,26 @@ pen & paper RPG game design".
 * Docker
 
 
+## User Interface
+
+The webpage contains short project description and two forms with instructions.
+
+![empty-forms](./readme_images/empty-forms.png)
+
+In the manual-input form, every field has question mark sign next to it. Hovering over it displays a prompt showing the acceptable range of values.
```

Comment:
To nie jest prompt ;)  information, message, ...?

Reply:
\+ warto wspomnieć, że jest też formularz na JSONy i po jego submicie odczytane wartości zostaną zamieszczone w tym manual-input, żeby się ktoś nie zdziwił